app/metrics.yaml

```diff
@@ -8147,28 +8147,88 @@ wallpapers:
   wallpaper_selected:
     type: event
     description: |
-      A wallpaper is selected from the settings screen.
+      A wallpaper is selected from the settings screen or onboarding dialog.
     extra_keys:
       name:
         description: The name of the selected wallpaper
         type: string
       theme_collection:
         description: The theme collection the selected wallpaper belongs to.
         type: string
+      source:
+        description: |
+          A string that tells us how the user selected the wallpaper.
+          Possible values are: `settings`, `onboarding`
+        type: string
     bugs:
       - https://github.com/mozilla-mobile/fenix/issues/23381
     data_reviews:
       - https://github.com/mozilla-mobile/fenix/pull/23382
+      - https://github.com/mozilla-mobile/fenix/pull/26893
     notification_emails:
       - android-probes@mozilla.com
     data_sensitivity:
       - interaction
-    expires: 114
+    expires: 116
     no_lint:
       - COMMON_PREFIX
     metadata:
       tags:
         - Wallpapers
+  onboarding_opened:
+    type: event
+    description: |
+      The wallpaper onboarding dialog has been displayed.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/26215
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/26893
+    notification_emails:
+      - android-probes@mozilla.com
+    data_sensitivity:
+      - interaction
+    expires: 116
+    metadata:
+      tags:
+        - Wallpapers
+  onboarding_closed:
+    type: event
+    description: |
+      The wallpaper onboarding dialog has been closed.
+    extra_keys:
+      is_selected:
+        description: Weather or not a new wallpaper has been selected.
+        type: boolean
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/26215
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/26893
+    notification_emails:
+      - android-probes@mozilla.com
+      - mavduevskiy@mozilla.com
+    data_sensitivity:
+      - interaction
+    expires: 116
+    metadata:
+      tags:
+        - Wallpapers
+  onboarding_learn_more_clicked:
+    type: event
+    description: |
+      The wallpaper onboarding learn more button was clicked.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/26215
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/26893
```

Comment:
I apparently just added this comment to a random commit instead of the PR, so apologies if you saw it already.

If this can be done without re-doing the data review request:

I think this should be renamed to explore more or whatever the text of the button is, in order to help differentiate it from the learn more URLs that will be included in seasonal collections on the settings page.

Reply:
@rocketsroger 
Should I do a separate data review request for it?

Reply:
could change it to `onboarding_explore_more_click`, there is a limit for chars

Reply:
Please update the data-review, I'll delete my review and redo it.  Thanks

Reply:
@rocketsroger updated!